src/webhook_launcher/settings.py

```diff
@@ -142,6 +149,32 @@
         "last_name": lname_attr,
         "email": mail_attr,
     }
+
+    # Need to specify ldap_group_search_dn for any of this to work
+    if config.has_option('ldap', 'ldap_group_search_dn'):
+        AUTH_LDAP_GROUP_SEARCH = LDAPSearch(
+            config.get('ldap', 'ldap_group_search_dn', raw=True),
+            ldap.SCOPE_SUBTREE, "(objectClass=group)")
+
+        # Dynamic import of a suitable group type
+        MY_LDAP_GROUP_TYPE = "NestedActiveDirectoryGroupType"
+        if config.has_option('ldap', 'ldap_group_type'):
+            MY_LDAP_GROUP_TYPE = config.get('ldap', 'ldap_group_type', raw=True)
+        dal_module = __import__("django_auth_ldap.config", globals(), locals(),
+                            [MY_LDAP_GROUP_TYPE])
+        MyLDAPGroupType = getattr(dal_module, MY_LDAP_GROUP_TYPE)        
```

Comment:
Yuck.  So this django_auth_ldap.config supports importing arbitrary symbols for the group type, but if someone sets that in the configuration file we will only know at runtime which symbol to import? Pff.